tests/posix_netdb/Makefile

```diff
@@ -0,0 +1,17 @@
+include ../Makefile.tests_common
+
+USEMODULE += embunit
+
+# make sure we have an implementation of sock_types.h
+USEMODULE += sock_udp
+USEMODULE += gnrc_ipv6
+
+USEMODULE += ipv4_addr
+USEMODULE += ipv6_addr
+
+# use mock data for the test
+USEMODULE += sock_dns_mock
+
+USEMODULE += posix_netdb
+
+include $(RIOTBASE)/Makefile.include
```

Comment:
```bash
make -C tests/posix_netdb/ all-valgrind term-valgrind
```

yields

```
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
--105511-- warning: evaluate_Dwarf3_Expr: unhandled DW_OP_ 0x9b
==105511== Invalid read of size 4
==105511==    at 0x804BC0C: ??? (in /home/mlenders/Repositories/RIOT-OS/RIOT/tests/posix_netdb/bin/native/tests_posix_netdb.elf)
==105511==  Address 0x806c790 is in the brk data segment 0x8079000-0x8078fff
==105511== 
{ "threads": [{ "name": "main", "stack_size": 12288, "stack_used": 2444}]}
```

Sadly no line number but somewhere is an out of bounds read error to one of the `malloc`'d memory areas in this test (or the netdb code).